frontend/src/components/cart-page/CartSummarySection.tsx

```diff
@@ -0,0 +1,72 @@
+import { useContext, useMemo } from 'react'
+import { useNavigate } from 'react-router-dom';
+
+import { ShoppingCartContext } from '../../contexts/shopping-cart-context/shoppingCartContext';
+import { Box, Button, Card, CardContent, Divider, Typography } from "@mui/material";
+
+import cartStyles from './cartStyles';
+const {
+    totalCardStyle,
+    buttonContainerStyle,
+    buttonStyle,
+} = cartStyles;
+
+const CartSummarySection = () => {
+    const navigate = useNavigate();
+    const { cartItems, userSignin } = useContext(ShoppingCartContext);
+
+    const SHIPPING_PATH = "/shipping";
+    const SIGNIN_PATH = "/user/signin";
```

Comment:
Better if you place these in a constants folder.
i could find `/user/signin` in 8 files & `/shipping` in 4 💭